infrastructure-configs/demeter.json

```diff
@@ -3,14 +3,37 @@
 "substitutions": {
     "AOCS_HDFS_BAMS": "hdfs:///datasets/ovarian/ega-tim",
     "HG19_FASTA": "/hpc/users/ahujaa01/reference_genomes/hg19-reference/ucsc.hg19.fasta"
-}
+},
 
-"spark_configuration": {
-    "spark.driver.maxResultSize": "0",
-    "spark.yarn.executor.memoryOverhead": "6000",
-    "spark.executor.memory": "16g",
-    "spark.default.parallelism": "10000"
+"spark_submit_arguments": [
+    "--deploy-mode", "cluster",
+    "--master", "yarn",
+    "--executor-cores", "6",
+    "--executor-memory", "17g",
+    "--driver-memory", "10g",
 
-}
+    "--conf", "spark.driver.maxResultSize=0",
```

Comment:
I'd be ok to put the spark config in a spark config file but I'm not sure that addresses your concern. It would still be a file in our git repo with demeter-specific settings that people would have to change to run in a different environment. Hopefully these settings don't affect the called variants but they will affect the run time, and my hope is to use this repo to also benchmark speed.

The goal is definitely to have the benchmark be independent of cluster setup though. That's why these settings are in their own `demeter.json` file.